src/main/java/hudson/plugins/copyartifact/BuildSelectorParameter.java

```diff
@@ -90,6 +97,35 @@ public String getDisplayName() {
         public DescriptorExtensionList<BuildSelector,Descriptor<BuildSelector>> getBuildSelectors() {
             return Hudson.getInstance().<BuildSelector,Descriptor<BuildSelector>>getDescriptorList(BuildSelector.class);
         }
+
+        /**
+         * @return {@link BuildSelector}s available for BuildSelectorParameter.
+         */
+        public List<Descriptor<BuildSelector>> getAvailableBuildSelectorList() {
+            return Lists.newArrayList(Collections2.filter(
+                    Jenkins.getInstance().getDescriptorList(BuildSelector.class),
+                    new Predicate<Descriptor<BuildSelector>>() {
+                        public boolean apply(Descriptor<BuildSelector> input) {
+                            return !"ParameterizedBuildSelector".equals(input.clazz.getSimpleName());
+                        };
+                    }
+            ));
+        }
+        
+        @Override
+        public String getHelpFile(String fieldName) {
+            if ("defaultSelector".equals(fieldName) || "parameter".equals(fieldName)) {
+                // Display the help file of `Copyartifact#getSelector` ("which build" field)
+                // for `defaultSelector` ("Default Selector" field) in project configuration pages
+                // and the value of build parameter ("Build selector for Copy Artifact" field)
+                // in "This build requires parameters" pages.
+                Descriptor<?> d = Jenkins.getInstance().getDescriptor(CopyArtifact.class);
+                if (d != null) {
+                    return d.getHelpFile("selector");
+                }
+            }
+            return super.getHelpFile(fieldName);
+        }
```

Comment:
This displays the same help for `Copyartifact#getSelector` and `BuildSelectorParameter#getDefaultSelector` and the value in "The build requires parameters" page.